pyglmnet/pyglmnet.py

```diff
@@ -161,6 +167,17 @@ def _grad_L2loss(distr, alpha, Tau, reg_lambda, X, y, eta, beta):
             + reg_lambda * (1 - alpha) * \
             np.dot(InvCov, beta[1:])
 
+    elif distr == 'probit':
+        prob = probit(z)
+        grad_prob = grad_probit(z)
+        grad_beta0 = -1. / n_samples * \
+            np.sum((y * (grad_prob / prob)) -
+                   ((1 - y) * (grad_prob / (1 - prob))))
+        grad_logl = ((y * (grad_prob / prob)) -
+                     ((1 - y) * (grad_prob / (1 - prob))))
+        grad_beta = -1. / n_samples * np.transpose(np.dot(grad_logl.T, X)) + \
```

Comment:
you don't need the `np.transpose(...)`. It's basically same as doing `np.dot(X.T, grad_logl)`